### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by radhikaj**

### CODE REVIEW

**Review by mingweishih — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,18 +24,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+## Changed
+- Updated libcxx to version 10.0.1
+
+
+[v0.17.6][v0.17.6_log]
+--------------
+
+### Added
+- Added support FIPS-enabled OpenSSL based on [SymCrypt](https://github.com/Microsoft/SymCrypt)
```
Comment (CyanDevs) [inline CHANGELOG.md:36]: Nit: Added support `for` FIPS-enabled OpenSSL based on [SymCrypt](https://github.com/Microsoft/SymCrypt)`.`

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,18 +24,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 ### Security
 - Update mbedTLS to version 2.16.12. Refer to the [2.16.12](https://github.com/ARMmbed/mbedtls/releases/tag/v2.16.12) release notes for the set of issues addressed.
+- Note: 2.16 LTS is at End Of Life. mbedTLS libs included with the Open Enclave SDK will move to use the 2.28 LTS branch in the next release. 2.28.0 has certain breaking changes. To understand how these changes will impact your application, please refer to the release notes for [2.28.0](https://github.com/ARMmbed/mbedtls/releases/tag/v2.28.0).
+
+[v0.17.5][v0.17.5_log]
+--------------
+
+### Added 
+- Added MUSL time functions
+asctime, asctime_r, ctime, ctime_r, ftime, localtime, localtime_r, strptime, timespec_get, wcsftime
```
Comment (CyanDevs) [inline CHANGELOG.md:54]: Nit: ending `.`

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,18 +24,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Changed
+- Fix bug with incorrect layout of thread-local sections (tbss and tdata). Previous releases of OE had a bug where these sections
```
Comment (CyanDevs) [inline CHANGELOG.md:57]: Nit: `Fixed` bug...

**Review by CyanDevs — APPROVED** [review]

**Pull request merged**